node/resolvers/checkout/index.ts

```diff
@@ -56,6 +57,18 @@ interface AddItemArgs {
   items?: OrderFormItemInput[]
 }
 
+interface SkuPickupSlAArgs {
+  itemId: string
+  seller: string
+  lat: string
+  long: string
+  country: string
+}
+
+interface SkuPickupSLAArgs extends SkuPickupSlAArgs {
```

Comment:
I did not understand the need for these two interfaces, why are they split into two? And if they do need to be split, I think it would be a better idea to rename one of them, since they basically have the same name.